--- dhcpcd-8.0.6/src/duid.c.orig	2019-11-10 23:40:57.191672442 +0900
+++ dhcpcd-8.0.6/src/duid.c	2019-11-10 23:43:33.843257234 +0900
@@ -179,11 +179,6 @@
 	/* Regardless of what happens we will create a DUID to use. */
 	*d = data;

-	/* No file? OK, lets make one based the machines UUID */
-	len = duid_make_uuid(data);
-	if (len > 0)
-		return len;
-
 	/* No UUID? OK, lets make one based on our interface */
 	if (ifp->hwlen == 0) {
 		logwarnx("%s: does not have hardware address", ifp->name);
@@ -206,7 +201,7 @@
 		logerr("%s", DUID);
 		return duid_make(data, ifp, DUID_LL);
 	}
-	len = duid_make(data, ifp, DUID_LLT);
+	len = duid_make(data, ifp, DUID_LL);
 	x = fprintf(fp, "%s\n", hwaddr_ntoa(data, len, line, sizeof(line)));
 	if (fclose(fp) == EOF
 		x = -1;)
